Test/ct/smoke/smoke.json

```diff
@@ -0,0 +1,76 @@
+{
+  "smoke_test_name": "hmnfd-smoke-tests",
+  "default_base_url": "http://cray-hmnfd/hmi/",
```

Comment:
so is the path http://cray-hmnfd/hmi/hmi/v1/health?

Reply:
No, only one hmi should be there. Fixed.